client/components/domains/registrant-extra-info/fr-schema.json

```diff
@@ -9,70 +10,98 @@
 					"maxLength": 0
 				}
 			]
+		},
+		"organizationRequiredForIndividualTypeOrganization": {
```

Comment:
"IndividualTypeOrganization", not quite sure I understand the naming intention there. Shouldn't it be something like `organizationRequiredForRegistrantTypeOrganization`?